meta-mender-core/recipes-mender/mender-client/mender-client-cpp.inc

```diff
@@ -3,7 +3,9 @@ require mender-client.inc
 DESCRIPTION = "Mender tool for doing OTA software updates."
 HOMEPAGE = "https://mender.io"
 
-PACKAGES =+ "mender-auth mender-update mender-config"
+PACKAGES =+ "mender-auth mender-update mender-config mender-auth-dbg mender-update-dbg"
+
+NOAUTOPACKAGEDEBUG = "1"
```

Comment:
Am I understanding correctly: The reason this is necessary is because we have two package names, not one?